lib/pages/home/modals/save_community.dart

```diff
@@ -173,8 +173,10 @@ class OBSaveCommunityModalState extends State<OBSaveCommunityModal> {
                                 size: OBTextFormFieldSize.medium,
                                 controller: _titleController,
                                 decoration: InputDecoration(
-                                  labelText: _localizationService.community__save_community_label_title,
-                                  hintText: _localizationService.community__save_community_label_title_hint_text,
+                                  labelText: _localizationService
+                                      .community__save_community_label_title,
+                                  hintText: _localizationService
+                                      .community__save_community_label_title_hint_text,
```

Comment:
It seems your editor auto-formatted the code based on the line max length, please return these changes so we can avoid future conflicts with the main Okuna repository.

If you are using VSCODE, please adding the following to `.vscode/settings.json` should avoid this issue:

```json
    "[dart]": {
        "editor.formatOnSave": false
    }
```